index.rst

```diff
@@ -6,24 +6,233 @@
+
+Right image: EUI screenshot showing drives 31 and 32 enabled when the rest are disabled. The opposite behaviour is also observed (31,32 disabled when the rest are enabled)
+
+Azimuth torque profiles
+=======================
+Finally, we show the torque profiles for 4 pairs of azimuth slews run at 5% of the designed maximum velocity. These show the low hysteresis of the system for azimuth slews, with fairly little dependence on the telescope elevation.
+
+.. figure:: ./_static/azimuth_slews_20230630.png
+   :name: azimuth-slews
+
+   Here we show the required torque as a function of azimuth angle over the range of 70-250 *deg*. The color of the line denotes the type of slew (positive/negative) and the system state (telescope elevation=90/0).
+
+Open issues to be fixed/investigated
+====================================
+It is not possible to balance the telescope within the TMA requirements with the present situation.
+This increases the risk of a dangerous unintended motion and produces unnecessary image degrading heat.
+The torque hysteresis will also likely produce difficulties in pointing and tracking.
+
+1. It should be verified that the torque reversal near horizon pointing is produced by the premature truncation of the elevation magnets. If corrective action is shown to be impractical, the software interlocks should be modified to allow operation of the balancing units without the elevation pins inserted horizon pointing, and a new procedure produced for balancing the elevation assembly with the equipment available.
+
+2. The cause of the torque jump near zenith pointing should be determined and eliminated if possible. If not the software interlocks should be modified to allow operation of the balancing units without the elevation pins inserted zenith pointing.
+
+3. The cause of the torque hysteresis should be determined and eliminated. As a result of the EUI it is possible that magnets 31 and 32 are operating incorrectly or reporting their torques (current) incorrectly. If this deficiency is not related to the torque hysteresis it should be corrected regardless. The +x and –x should be labelled correctly. If the large torque hysteresis is produced by another phenomenon this still needs to be determined and rectified. The telescope cannot be properly operated under these conditions.
+
+
+
+.. _appendix-possible-causes:
+
+Appendix: other considered causes of the torque behavior
+=========================================================
+
+Elevation breaks
+----------------
+
+We checked for possible contact of the elevation brakes and the TMA during slews causing a dragging effect.
+This was done in a few different ways.
+First, by looking at the wear patterns between the break pads and the elevation axis at different elevations, there seemed to be some variance in the wear but nothing obvious (see images below or `SUMMIT-7581`).
+Then, we took infrared images of the TMA during slews no hot points were detected.
```

Comment:
Suggestion: "Then, we took infrared images of the TMA during slews. No hot points were detected."